manage.py

```diff
@@ -98,6 +100,12 @@ def init_db(skip_create_db=False, test_db=False):
     """
     click.echo("Initializing the database...")
 
+    if force:
+        click.echo("Dropping Database")
```

Comment:
This needs to be fixed too. Also, this is a temporary fix. The database and the user must be dropped using the `postgres` user (as in LB).